lldb/source/Plugins/ObjectFile/ELF/ObjectFileELF.cpp

```diff
@@ -2067,10 +2110,8 @@ unsigned ObjectFileELF::ParseSymbols(Symtab *symtab, user_id_t start_id,
   SectionList *module_section_list =
       module_sp ? module_sp->GetSectionList() : nullptr;
 
-  // Local cache to avoid doing a FindSectionByName for each symbol. The "const
-  // char*" key must came from a ConstString object so they can be compared by
-  // pointer
-  std::unordered_map<const char *, lldb::SectionSP> section_name_to_section;
+  // Cache the section mapping
```

Comment:
Nit: missing period. A map from SectionSP to SectionSP looks somewhat suspicious, might be worth explaining what the key and the value mean.

Reply:
I'll improve the comment.